app/models/runtime/revision_sidecar_process_type_model.rb

```diff
@@ -6,11 +6,19 @@ class RevisionSidecarProcessTypeModel < Sequel::Model(:revision_sidecar_process_
                 key: :revision_sidecar_guid,
                 without_guid_generation: true
 
+    def around_save
+      yield
+    rescue Sequel::UniqueConstraintViolation => e
+      raise e unless e.message.include?('revision_sidecar_process_types_revision_sidecar_guid_type_index')
+
+      errors.add(%i[revision_sidecar_guid type], "Sidecar is already associated with process type #{type}")
```

Comment:
Would it make sense to print different error messages for "revision" sidecars? (I guess not...)